tests/lib/prepare-project.sh

```diff
@@ -35,10 +35,14 @@ build_deb(){
 download_from_ppa(){
     local ppa_version="$1"
 
-    for pkg in snapd; do
+    for pkg in snapd snap-confine ubuntu-core-launcher; do
```

Comment:
Maybe a small comment why these extra dependencies (that are not needed in versions > 2.23) are downloaded here?